src/clients/BundleDataClient.ts

```diff
@@ -173,6 +177,7 @@ export class BundleDataClient {
     if (!isDefined(this.clients?.arweaveClient)) {
       return undefined;
     }
+    const start = Date.now();
```

Comment:
Matt previously identified that performance.now() is preferable because it uses a monotonic clock.

```suggestion
    const start = pperformance.now();
```